README.md

```diff
@@ -22,14 +22,18 @@ The API is a separate project and not part of this library.
 
 ## Configuration
 https://github.com/acheong08/ChatGPT/wiki/Setup
+
+Example:
 ```json
 {
-  "session_token": "<token>",
+  "email": "<EMAIL>",
+  "password": "<PASSWORD>",
+  "captcha": "<2CAPTCHA_API_KEY>",
```

Comment:
Might want session token to still be optional. Not everyone has a 2Captcha key.

Reply:
I realize there are a lot more options in setup.md. Might to also show them here. 

I realize session token is still available